Gems/Atom/RPI/Code/Source/RPI.Public/RPIUtils.cpp

```diff
@@ -110,6 +110,26 @@ namespace AZ
                 return ((value - origMin) / (origMax - origMin)) * (scaledMax - scaledMin) + scaledMin;
```

Comment:
Maybe add assert to make sure value >= origMin and value >= origMax. Otherwise you would need to trim it.

Reply:
sounds good.